.readthedocs.yml

```diff
@@ -12,7 +12,7 @@ build:
     - cmake
   tools:
     python: "3.11"
-    rust: "1.64"
+    rust: "1.70"  # RDT does not offer our minimal version 1.65
```

Comment:
Should we just use the latest rustc version on rtd?

Reply:
ok, this is the latest version. I would just remove the comment about the minimal version then.

Reply:
```suggestion
    rust: "1.70"
```